code/src/layouts/app-footer/AppFooter.scss

```diff
@@ -0,0 +1,64 @@
+@import "@/styles/scss/foundation/all";
+
+.footer {
+  padding: spacing(18) 0;
+  background-color: $white;
```

Comment:
Use conbination of `scss` variables and `css` variables. Check for example `AppButton.scss`

Reply:
Thanks, I made it here and in other places the same way as in `AppButton.scss`.